lib/l10n/app_de.arb

```diff
@@ -259,8 +259,10 @@
 
   "widgetTripBuilderTabUpcoming": "Zukünftig",
   "widgetTripBuilderTabPast": "Vergangen",
-  "widgetTripBuilderNoUpcoming": "Keine zukünftigen {name}.",
-  "widgetTripBuilderNoPast": "Keine vergangenen {name}.",
+  "widgetTripBuilderNoUpcomingDrives": "Keine zukünftigen Fahrten.",
```

Comment:
I would remove the '.' at the end. We put the '.' for sentences but I would not use them for hints or small infos

Reply:
If you would do 'Du hast keine ...' then the '.' can be there in my opinion

Reply:
I removed the dot.

@Eddie-42 shrug would be even better, right?